Lib/ufo2ft/featureWriters/markFeatureWriter.py

```diff
@@ -3,13 +3,11 @@
 from collections import OrderedDict, defaultdict
 from functools import partial
 
-from fontTools.misc.fixedTools import otRound
-
 from ufo2ft.constants import INDIC_SCRIPTS, USE_SCRIPTS
 from ufo2ft.featureWriters import BaseFeatureWriter, ast
 from ufo2ft.util import (
     classifyGlyphs,
-    quantize,
```

Comment:
@simoncozens I had to change glyphsLib's markFeatureWriter to import `quantize` from `ufo2ft.util` otherwise we get ImportError with this PR:
https://github.com/googlefonts/glyphsLib/commit/ae08b26716188d027de9c70fb9dc7c017f868dad

Reply:
oh I had forgotten that you already have a PR for that, sorry: https://github.com/googlefonts/glyphsLib/pull/957/

ok so we shall merge that one once we release this